app/component/itinerary/ItineraryDetails.js

```diff
@@ -687,7 +347,346 @@ const withRelay = createFragmentContainer(
         }
       }
     `,
-  },
+    itineraryRef,
+  );
+
+  const shouldShowDisclaimer =
+    config.showDisclaimer &&
+    match.params.hash !== streetHash.walk &&
+    match.params.hash !== streetHash.bike;
+
+  function getFutureText(startTime) {
+    const refTime = Date.now();
```

Comment:
Move getFutureText  outside component body. No need to redefine at every render.